src/components/Filter.spec.js

```diff
@@ -1,17 +1,16 @@
 import { render, screen } from '@testing-library/react';
-import FilterNotes from './FilterNotes';
+import Filter from './Filter';
 
-describe('FilterNotes', () => {
-  it('renders five buttons', () => {
-    render(<FilterNotes />);
+describe('Filter', () => {
+  it('renders four buttons', () => {
+    const categories = ['family', 'friends', 'vacation', 'others'];
+    render(<Filter categories={categories} />);
 
-    const button1 = screen.getByRole('button', { name: /all/i });
     const button2 = screen.getByRole('button', { name: /family/i });
```

Comment:
You can take a shorter version here.
`expect(screen.getAllByRole('button').length.toBe(4)`
P.S.: May I've forget a bracket